src/conn/conn_log.c

```diff
@@ -163,6 +163,9 @@ __logmgr_config(
 			FLD_SET(conn->log_flags, WT_CONN_LOG_RECOVER_ERR);
 	}
 
+	WT_RET(__wt_config_gets(session, cfg, "log.trickle", &cval));
+	conn->log_trickle = (uint32_t)cval.val;
```

Comment:
It's not just the user-visible name I don't really like ``trickle`` for, I'd quite like for a name change to flow down.